module_presentations.py

```diff
@@ -4014,7 +4038,7 @@ def prsnt_generate_find_object_slot():
         (gt, "$g_target_player_id", 0),
         (try_begin),
           (player_is_active, "$g_target_player_id"),
-          (player_slot_eq, "$g_target_player_id", slot_player_list_button_id, "$g_target_player_overlay_id"),
+          (player_slot_eq, "$g_target_player_id", slot_player_list_button_id, "$g_target_player_overlay_id"), # bug caused here

```

Comment:
This seems to be the cause of the bug.

When the Private Message chat is opened without the player list being made to open first it seems to cause this line to fail and therefore cause the `NO STRING!` message on the chatbox.